senders/msteams/msteams.go

```diff
@@ -0,0 +1,207 @@
+package msteams
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/moira-alert/moira"
+	"github.com/russross/blackfriday/v2"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+)
```

Comment:
Imports are not sorted properly here. I don't know why our linter misses this :-(

Please, sort imports using `goimports` (https://godoc.org/golang.org/x/tools/cmd/goimports).